crates/evm/src/trace/decoder.rs

```diff
@@ -144,25 +140,25 @@ impl CallTraceDecoder {
             // TODO: These are the Ethereum precompiles. We should add a way to support precompiles
             // for other networks, too.
             precompiles: precompiles!(
-                0x01: ecrecover(hash: FixedBytes(32), v: Uint(256), r: Uint(256), s: Uint(256)) -> (publicAddress: Address),
```

Comment:
This change looks a bit odd cc @DaniPopes

Reply:
The whole point of this macro is to make it easy to modify. Slapping `format` everywhere doesn't really help towards that. Anyway I am working on removing this in #5998 since some precompiles don't adhere to ABI so it doesn't matter.